assets/csidriveroperators/ibm-vpc-block/08_deployment.yaml

```diff
@@ -18,6 +18,9 @@ spec:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
       labels:
         name: ibm-vpc-block-csi-driver-operator
+        openshift.storage.network-policy.api-server: allow
+        openshift.storage.network-policy.dns: allow
+        openshift.storage.network-policy.operator-metrics-range: allow
```

Comment:
Similar to vsphere, I think you need `openshift.storage.network-policy.all-egress: allow` here to allow cloud API calls from [secretsync.go](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/main/pkg/controller/secret/secretsync.go). Error from [e2e-ibmcloud-csi](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cluster-storage-operator/599/pull-ci-openshift-cluster-storage-operator-main-e2e-ibmcloud-csi/1947448182573109248):
`level=error msg=IBMVPCBlockCSIDriverOperatorCRDegraded: SecretSyncDegraded: An error occurred while performing the 'authenticate' step: Post "https://iam.cloud.ibm.com/identity/token": context deadline exceeded (Client.Timeout exceeded while awaiting headers)`